ui/src/app/views/profile-edit/component.css

```diff
@@ -1,3 +1,25 @@
 h2 {
   display: inline-block;
 }
+
+fieldset {
```

Comment:
It may make sense to avoid the use of the word fieldset. We've considered, and plan to eventually, have a concept of a fieldset in our UI, which is some user specified sets of fields they are interested in for a cohort(s), so I have a bit of concern about this getting confusing down the line.